Manual.md

```diff
@@ -1444,9 +1444,25 @@ Python packages should be built with the `python{,2,3}-module` build style, if p
 This sets some environment variables required to allow cross compilation. Support to allow
 building a python module for multiple versions from a single template is also possible.
 
-To allow cross compilation, the `python-devel` package (for python 2.7) must be added
-to `hostmakedepends` and `makedepends`. If any other python version is also supported,
-for example python3.4, those must also be added as host and target build dependencies.
+If a python package includes a compiled extension, the `python-devel` (for python 2.7) or
+`python3-devel` (for python 3.X) packages should be added to `makedepends`. Generally,
+dependencies on `python-*` or `python3-*` packages should be included in `hostmakedepends` because
+the package build process often requires the host interpreter be run with the necessary packages
+available. In most cases, especially those with pure-python dependencies, it is not necessary to
```

Comment:
Can we make it clear that these should be the same dependencies that would also be in `depends`? I think that's what's needed, right?

Reply:
The `setuptools` installer recognizes `install_requires`, which should generally be placed in `hostmakedepends`, and `requires`, which should be placed in `depends`. I'm going to rewrite some of this based on comments from @Chocimier to simplify and clarify this point.